db/models/Recipe.js

```diff
@@ -3,13 +3,20 @@ import mongoose from "mongoose";
 const { Schema } = mongoose;
 
 const recipeSchema = new Schema({
-  title: { type: String, required: true },
+  /*   id: { type: String },
+   */ title: { type: String, required: true },
```

Comment:
you should remove all commented out code before creating a PR

```suggestion
title: { type: String, required: true },
```